pkg/internal/checkup/checkup.go

```diff
@@ -241,6 +252,19 @@ func (c *Checkup) waitForVMIDeletion(ctx context.Context) error {
 	return nil
 }
 
+func (c *Checkup) cleanupVMI() {
+	const setupCleanupTimeout = 30 * time.Second
```

Comment:
Are you sure 30s is sufficient timeout? from my experience it might take longer.
I would crank it up to 2 minutes, also for the pod context timeout

Reply:
It is what we [use](https://github.com/kiagnose/kiagnose/pull/239) in the VM latency checkup.
In case we will see that it won't suffice, we can increase it.
We have another line of defense - the checkup pod owns the VMI and traffic generator pod, so when it gets deleted, they are garbage-collected.